packages/@glimmer/runtime/test/partial-test.ts

```diff
@@ -30,7 +30,7 @@ function commonSetup() {
 function render(template: Template, context = {}) {
   self = new UpdatableReference(context);
   env.begin();
-  let cursor = { element: root, nextSibling: null };
+  let cursor = { element: root, nextSibling: null } as Cursor;
```

Comment:
is Cursor an interface or is it a class? `let cursor: Cursor = { element: root, nextSibling: null }`

Reply:
It's actually a class for non-obvious reasons, but used like an interface almost everywhere. The cast is necessary because `root` is an `HTMLElement` and needs to be explicitly cast to the expected `Simple.Element` type.